src/main/kotlin/Creatable.kt

```diff
@@ -0,0 +1,4 @@
+interface Creatable {
+    fun createObject() {}
```

Comment:
🍏 Если не требуется дефолтная реализация, то можно оставить так:
```suggestion
    fun createObject()
```